.github/workflows/release.yml

```diff
@@ -0,0 +1,70 @@
+name: Build Releases
+
+on:
+  release:
+    types: [published]
+  push:
+    branches:
+      - 'master'
+      - 'develop'
```

Comment:
What is develop doing in here? Sorry I was lazy and didn't look up the docs around this...

Reply:
See the updated main comment above. This is Option (1), creating Assets, which won't go into the Releases section.

We can remove this, but it's designed to allow non-devs to quickly test the latest Develop or Master without needing to compile code.

Reply:
I think it's a cool idea, but let's remove it for now and add it back in in the future if there's ever any demonstrated need for it.

Reply:
If I could add another reason to keep this: (1) it runs faster than the tests, and runs in parallel, so there are no downsides, and (2) it allows regular testing of this workflow, instead of only running the workflow on releases.

I'm a little afraid it will become stale if we only run it on release creation.

If you still think I should remove it, I'm happy to.

Reply:
Some thoughts:
1. I could skip the artifact upload if that's an issue
2. Or, decrease the lifespan to 5 days

But, for public repos, [my understanding](https://github.com/orgs/community/discussions/26438) is that there is no storage limit for Github Actions.

Again, no strong opinions here, but given that it's free in terms of both money and time, I don't see downsides.

Reply:
If it's totally free, sure.

Reply:
Appreciate that, @artoonie. Sounds good re: the draft method.

@tarheel haha I knew that was coming! Probably relatively low. In all seriousness though, my main motivation for working on this project is the hope that it will one day unblock solutions to the climate crisis. I was also brain damaged by doing lots of LCA work in the past, so these are the things that keep me up at night.